server/controllers/AutomationController.js

```diff
@@ -25,130 +26,157 @@ const axiosConfig = {
     timeout: 1000
 };
 
-const scheduleNewJob = async (req, res) => {
-    try {
```

Comment:
I was getting ready to mention that Express 4 doesn't recognize rejected promises, so you can imagine my surprise to learn that [this project relies on a pre-release version of the framework](https://github.com/w3c/aria-at-app/commit/dc4edc33a)!

Reply:
Yeah I checked for Express 5 before I made these changes.

Reply:
You also have the test coverage to prove it :) I didn't mean to suggestion otherwise--I was the one caught off-guard.